peer-review-bundle-spec.md

```diff
@@ -0,0 +1,147 @@
+# Qwiklabs PeerReview Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it
+easier to see the general file structure.
+
+```yaml
+entity_type: PeerReview
+schema_version: 1
+
+default_locale: en
+
+title:
+  locales:
+    en: Create storyboards for your portfolio project
+
```

Comment:
Generally speaking, `dictionary_locales` are specific to the API bundle spec, not the *GitHub* bundle spec. It could be useful to establish this distinction early on to avoid confusion later on.

Reply:
This applies to similar instances below as well.

Reply:
do you mind give more info about this:
do you mean have:
"title: Create storyboards for your portfolio project"
is good enough in GitHub?

because the examples I see from lab, course, exam, are
title:
    locales:
        en:

course: https://screenshot.googleplex.com/9n2g4vppVuLyMDb
lab: https://screenshot.googleplex.com/BoxZ9rkbwG3NBvp
exam: https://screenshot.googleplex.com/AZa62Xoaqo246Nd
resource: https://screenshot.googleplex.com/83QyxdoDEV5crg7

Reply:
Yes, these examples were are basically for the Unbunder API spec, not GitHub. GitWhisperer has the job of translating plain strings from different locale-specific yaml files into a dictionary locale in just one file.

Reply:
I think we should make this doc in GW spec, and change file name to "github-peer-review-bundle-spec", similar to lab. It makes it less confusing for authors. We can move this version to `api` folder.

Reply:
done.